pvacseq/lib/input_file_converter.py

```diff
@@ -297,3 +304,88 @@ def execute(self):
+            'start 3p',
+            'end 3p',
+            'name of fusion',
+            'tier of fusion',
+            'strand 5p',
+            'strand 3p',
+            'quantitation',
+            'is canonical boundary',
+            'can be in-frame',
+            'peptides',
+            'fusion positions',
+            'number of nucleotides in the break',
+            'transcripts',
+            'is canonical intron dinucleotide',
+        ]
+
+    def execute(self):
+        reader = open(self.input_file, 'r')
+        csv_reader = csv.DictReader(reader, delimiter='\t', fieldnames=self.input_fieldnames())
+        writer = open(self.output_file, 'w')
+        tsv_writer = csv.DictWriter(writer, delimiter='\t', fieldnames=self.output_headers())
+        tsv_writer.writeheader()
+        transcript_count = {}
+        for entry in csv_reader:
+            output_row = {
+                'chromosome_name'            : entry['chr 5p'],
+                'start'                      : entry['start 5p'],
+                'stop'                       : entry['end 5p'],
+                'reference'                  : 'fusion',
+                'variant'                    : 'fusion',
+                'gene_name'                  : entry['name of fusion'],
+                'amino_acid_change'          : 'NA',
+                'ensembl_gene_id'            : 'NA',
+                'protein_position'           : 'NA',
+                'amino_acid_change'          : 'NA',
+                'transcript_expression'      : 'NA',
+                'gene_expression'            : 'NA',
+                'normal_depth'               : 'NA',
+                'normal_vaf'                 : 'NA',
+                'tdna_depth'                 : 'NA',
+                'tdna_vaf'                   : 'NA',
+                'trna_depth'                 : 'NA',
+                'trna_vaf'                   : 'NA',
+            }
+
+            count = 1
+            for (fusion_position, transcript_set, fusion_amino_acid_sequence) in zip(entry['fusion positions'].split(','), entry['transcripts'].split(','), entry['peptides'].split(',')):
+                (five_p_transcripts, three_p_inframe_transcripts, three_p_frameshift_transcripts) = transcript_set.split(';')
+                inframe_fusions    = []
+                frameshift_fusions = []
+                for five_p_transcript in five_p_transcripts.split('|'):
+                    if len(three_p_inframe_transcripts):
+                        for three_p_inframe_transcript in three_p_inframe_transcripts.split('|'):
+                            inframe_fusions.append("%s-%s" % (five_p_transcript, three_p_inframe_transcript))
+                    if len(three_p_frameshift_transcripts):
+                        for three_p_frameshift_transcript in three_p_frameshift_transcripts.split('|'):
+                            frameshift_fusions.append("%s-%s" % (five_p_transcript, three_p_frameshift_transcript))
```

Comment:
Feel free to ignore this, but I wanted to at least come up with a version of combining these that made sense 😄

This could be:
```python
def fusions_for_three_p_transcripts(self, five_p_transcript, three_p_transcripts):
    fusions = []
    if len(three_p_transcripts):
        for three_p_transcript in three_p_transcripts.split('|'):
            fusions.append("%s=%s"% (five_p_transcript, three_p_transcript))
    return fusions
```
and then:
```python
  for five_p_transcript in five_p_transcripts.split('|'):
      inframe_fusions.extend(self.fusions_for_three_p_transcripts(five_p_transcript, three_p_inframe_transcripts))
      frameshift_fusions.extend(self.fusions_for_three_p_transcripts(five_p_transcript, three_p_frameshift_transcripts))
```

I had a somewhat different version of this that uses `map` and `lambda`, but I've read that's "unpythonic".  A list comprehension could probably work, I guess?